client/app/bundles/KudosApp/components/Kudo.jsx

```diff
@@ -61,21 +82,49 @@ export default class Kudo extends React.Component {
   }
 
   render() {
+
+    const Edit = () => <button
+      type='button'
+      className="btn update-button edit-button"
+      onClick={this.makeEditable}>
+        Edit
+      </button>
+
+    const Save = () => <button
+      type='button'
+      className='btn save-button edit-button btn-primary'
```

Comment:
Same here — I think `btn-primary` is included in `save-button` so why does it need to be included here?